app/services/submit_applicant_service.rb

```diff
@@ -2,12 +2,18 @@ class SubmitApplicantService < BaseCfeService
   def call(cfe_assessment_id, session_data)
     estimate = ApplicantForm.from_session(session_data)
 
-    applicant = {
+    base_attributes = {
       date_of_birth: estimate.over_60 ? 70.years.ago.to_date : 50.years.ago.to_date,
       has_partner_opponent: false,
       receives_qualifying_benefit: estimate.passporting,
       employed: estimate.employed,
     }
+    applicant = if FeatureFlags.enabled?(:asylum_and_immigration)
+                  base_attributes.merge({ receives_asylum_support: session_data["asylum_support"],
+                                          receives_qualifying_benefit: session_data["asylum_support"] })
```

Comment:
I have used a combination solution to merge the "receives_asylum_support: asylum_support_form.asylum_support || false" if the flag is in place as otherwise it breaks a load of tests.